README.md

```diff
@@ -1 +1,27 @@
-# mini-project
\ No newline at end of file
+# mini-project
+hello microverse
+
+Built With
+html
+css
+Linters
+Author
+rejoice Evans
+
+GitHub: [@githubhandle] https://github.com/rmjspecial
+Twitter: rmjspecial
+LinkedIn: rejoice evans
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+Hat tip to anyone whose code was used
+Inspiration
+etc
+📝 License
+This project is MIT licensed.
```

Comment:
- [ ] Kindly make sure to use [this readme](https://github.com/microverseinc/readme-template/blob/master/README.md), and please be sure that you use its format instead of just copying the raw text without the styling. You can click on the "raw" button to get the raw text with the styling to use in your readme. Also, make sure to edit each section with information that better suits your project

Reply:
- modified readme file